compiler/src/main/scala/org/scalajs/nscplugin/GenJSCode.scala

```diff
@@ -6605,6 +6605,23 @@ abstract class GenJSCode[G <: Global with Singleton](val global: G)
   def isStaticModule(sym: Symbol): Boolean =
     sym.isModuleClass && !isImplClass(sym) && !sym.isLifted
 
+  def isAnonymousClass(sym: Symbol): Boolean = {
+    /* sym.isAnonymousClass simply checks if
+     * `name containsName tpnme.ANON_CLASS_NAME`
+     * so after flatten (which we are) it identifies classes nested inside
+     * anonymous classes as anonymous (n.b. local classes, see #4278).
```

Comment:
Was that supposed to be `e.g.`:
```suggestion
     * anonymous classes as anonymous (e.g., local classes, see #4278).
```
?